webpack/published-ts-components.js

```diff
@@ -92,9 +94,8 @@ const publishedTSComponents = {
   useIsOverflowing: "hooks/useIsOverflowing.ts",
   useMediaQuery: "hooks/useMediaQuery",
   useVibeMediaQuery: "hooks/useVibeMediaQuery",
-  useActiveDescendantListFocus: "hooks/useActiveDescendantListFocus"
-  // Don't remove next line
-  // plop_marker:published-hooks
+  useActiveDescendantListFocus: "hooks/useActiveDescendantListFocus",
+  useHover: "hooks/useHover/useHover.ts"
```

Comment:
Our standard is not to add suffix (I know we there are suffix in some files but it's also mistakes)
```suggestion
  useHover: "hooks/useHover/useHover"
```